### ISSUE TITLE
Resolving Android Dependencies Fails after import: Gradle Failed to fetch dependencies

### ISSUE THREAD

**Issue by NeuralEvolution** [playgameservices/play-games-plugin-for-unity#2908]
Im using unity 2019.4.1 as soon as i import GPGS(10.09) package it trys to resolve dependencies and fails with the following error

Reporting analytics data: /androidresolver/resolve/gradle/failed/fetch?unityVersion=2019.4.1f1&unityPlatform=WindowsEditor&installSource=unitypackage&version=1.2.153&scope= 'Android Resolver: Gradle Resolve: Tool Extraction Failed'
ERROR: Gradle failed to fetch dependencies.

Failed to run 'F:\Games_In_Dev\crazy Tanks\Temp\PlayServicesResolverGradle\gradlew.bat --daemon -b "F:\Games_In_Dev\crazy Tanks\Temp\PlayServicesResolverGradle\PlayServicesResolver.scripts.download_artifacts.gradle" "-PANDROID_HOME=D:/Unity/2019.4.1f1/Editor/Data/PlaybackEngines/AndroidPlayer\SDK" "-PTARGET_DIR=F:\Games_In_Dev\crazy Tanks\Assets\Plugins\Android" "-PMAVEN_REPOS=https://maven.google.com/;file:///F:/Games_In_Dev/crazy%20Tanks/Assets/GooglePlayGames/Editor/m2repository" "-PPACKAGES_TO_COPY=com.google.android.gms:play-services-ads:19.2.0;com.google.games:gpgs-plugin-support:0.10.09" "-PUSE_JETIFIER=0" "-PDATA_BINDING_VERSION=3.4.0"'
stdout:

stderr:

FAILURE: Build failed with an exception.

* Where:
Settings file 'F:\Games_In_Dev\crazy Tanks\Temp\PlayServicesResolverGradle\settings.gradle'

* What went wrong:
Could not compile settings file 'F:\Games_In_Dev\crazy Tanks\Temp\PlayServicesResolverGradle\settings.gradle'.
> startup failed:
  General error during semantic analysis: Unsupported class file major version 57
  
  java.lang.IllegalArgumentException: Unsupported class file major version 57

Really need to get this working so any advice as to why this is happening and how to fix it would be great

Note this also occurs when i manually call Force Resolve

**Comment by MinhLuongNeo** [comment]
I have the same problem, maybe the solution is to use the old version

**Comment by NeuralEvolution** [comment]
So I upgraded my unity version to 2019.4.1 at first I had the same issue so I deleted the unity gpgs files including the ones in plugins and the play store silver files. Reimported all my assets. Imported the latestest version of playstore resolver. I think it was 1.5.6 or something like that which has the new dependency manager. Then imported playstore as normal. Did the force resolve and everything works nicely now
